test/deploy_veba_knative_processor.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2020 VMware, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-2
+
+# Sample Shell Script to test deployment of VEBA w/Knative Processor
+
+OVFTOOL_BIN_PATH="/Applications/VMware OVF Tool/ovftool"
```

Comment:
assumes OSX?

Reply:
User would need to specify the path to their `ovftool` installation, this is merely an example. It also matches whats already there :)